src/pages/Login/Login.tsx

```diff
@@ -91,13 +118,31 @@ const Login = () => {
             autoComplete="off"
             className={styles.form}
           >
-            <Form.Item
-              label="Email"
-              name="email"
-              rules={[{ required: true, message: "Please input your email!" }]}
-            >
-              <Input />
-            </Form.Item>
+            {emailMode === 1 && (
+              <Form.Item
+                label="Email"
+                name="email"
+                rules={[
+                  { required: true, message: "Please input your email!" },
```

Comment:
If all these input fields have similar rules and messages, we should devise a function that takes few parameters and returns these rules instead of writing again and again.